@here/harp-mapview/lib/MapView.ts

```diff
@@ -746,20 +764,10 @@ export class MapView extends THREE.EventDispatcher {
      */
     private readonly m_rteCamera = new THREE.PerspectiveCamera();
 
-    private m_yaw = 0;
-    private m_pitch = 0;
-    private m_roll = 0;
     private m_focalLength = 0;
-    private m_targetDistance = 0;
     private m_targetGeoPos = GeoCoordinates.fromObject(MapViewDefaults.target!);
```

Comment:
Should `m_targetGeoPos` and/or `m_targetWorldPos` be part of the `ViewState`? They are also related to the position and attitude of the camera.